app/src/main/java/net/squanchy/injection/ContextModule.java

```diff
@@ -0,0 +1,21 @@
+package net.squanchy.injection;
+
+import android.content.Context;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module
+public class ContextModule {
```

Comment:
I can depend on any context (i `getApplicationContext()` inside my constructor), but this context is needed in the ApplicationComponent, so it can't be an activity. That is the reason for the ContextModule as a different module from ActivityContextModule.